query/stdlib/testing/testing.go

```diff
@@ -103,6 +103,14 @@ var FluxEndToEndSkipList = map[string]map[string]string{
 		"join":               "unbounded test",
 		"alignTime":          "unbounded test",
 		"histogram_quantile": "mis-named columns for storage",
+		"distinct":           "failing test",
+		"fill":               "failing test",
+		"histogram":          "failing test",
+		"unique":             "failing test",
+	},
+	"experimental/oee": {
```

Comment:
These are the tests that were failing and need to be investigated later right? The ones that you mentioned at standup?

Reply:
I guess there was no description of why they were failing before. I think some of these tests might have been failing because we didnt have the ability to test non-table stream results before. We can figure that out when we circle back to them.

Reply:
Yeah. What happened was we introduced new experimental versions of some functions, and brought their tests over, slightly tweaked. Unfortunately, those tests were _already_ skipped, and I think in the time between initially skipping them and now, they work entirely differently.